### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by bill7pearl**
## In this PR I have:
- Implemented the design from the sneak peek wireframes.
- Focused on making sure everything functions first with just basic HTML structuring without CSS styling.
- Used methods that I have created in "processing data in models".

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,10 @@
-<h1>Selected user</h1>
\ No newline at end of file
+<h1>Selected user</h1>
+<img src="<%= @user.photo %>" alt="profile_pic">
+<%= @user.name%>
+<%= @user.posts_counter%>
+<%= @user.bio%>
+<% @user.posts.each do |d|%>
```
Comment (brytebee) [inline app/views/users/show.html.erb]: This currently displays all the post the user may have but would not be very nice if the user has a huge number of posts. Kindly ensure it renders only the 3 most recent posts for efficiency.

**Hint** Call the recent post method you defined in user model on the user instance. For example `@user.three_latest_post.each`

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1 +1,17 @@
...
+    <div><span>comments:<%=@post.comments_counter %></span>
+    <span>likes:<%=@post.likes_counter%></span> </div> </div>
+
+
+<p><%= @post.text %></p>
+</div>
+
+<div>
+    <h2>comments</h2>
+<% @post.comments.each do |comment|%>
```
Comment (brytebee) [inline app/views/posts/show.html.erb]: Just as explained in the case of users, the post index should likewise render the 5 most recent comments as required.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @bill7pearl👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (bill7pearl) [reply app/views/users/show.html.erb]: done

File: app/views/posts/show.html.erb (hunk 2 above)
Comment (bill7pearl) [reply app/views/posts/show.html.erb]: done

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1 +1,16 @@
-<h1>All posts</h1>
\ No newline at end of file
+<h1>All posts</h1>
+<img class="img" src="<%= @user.photo %>" alt="profile_pic">
+<%=@user.name %>
+<span>number of posts:  <%= @user.posts_counter%> </span>
+
+<% @post.each do |post| %>
```
Comment (brytebee) [inline app/views/posts/index.html.erb:6]: Optional: Kindly implement similar changes here.

**Review by brytebee — APPROVED** [review]
Hi, @bill7pearl 👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great DB setup.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



##

###  **Optional Suggestions:**
- N/A

##

Good luck! 🎉 

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bill7pearl** [comment]
@brytebee Thanks

**Pull request merged**